testcases/kernel/syscalls/write/write04.c

```diff
@@ -208,32 +182,24 @@ void alarm_handler(void)
  * setup()
  *	performs all ONE TIME setup for this test
  */
-void setup(void)
```

Comment:
And also please get rid of useless comments like this one that are explaining the obvious.